.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ env:
 
 jobs:
   linux-gcc-psv-build-test-codecov:
-    name: Linux Build using gcc & tests & code coverage
+    name: Linux gcc build & tests & code coverage
```

Comment:
There doesn't seem to be any naming pattern here. I think we should find one. I propose to start first with the type of pipeline then with the OS and compiler, followed by additional steps, all separated by `/`.
E.g. `PSV / Linux gcc 5.4 / Tests / Code coverage`